apps/organisations/models.py

```diff
@@ -112,7 +114,23 @@ class Organisation(models.Model):
         verbose_name=_('Imprint'),
         help_text=_('Please provide all the legally '
                     'required information of your imprint. '
-                    'The imprint will be shown on a separate page.')
+                    'The imprint will be shown on a separate page.'),
+        blank=True
+    )
+    terms_of_use = RichTextField(
+        verbose_name=_('Terms of use'),
+        help_text=_('Please provide your terms of use.'),
+        blank=True
+    )
+    data_protection = RichTextField(
+        verbose_name=_('Data protection'),
+        help_text=_('Please provide your data protection information.'),
+        blank=True
+    )
+    netiquette = RichTextField(
+        verbose_name=_('Netiquette'),
+        help_text=_('Please provide your rules for online discussions.'),
+        blank=True
```

Comment:
I wasnt sure about the help text, should I ask Frede about them?

Reply:
About the name and helptext? Yes, please ask Frede! For the German and English text. Also I think in Englilsh it's something else and you also need to update the field name. :)
And if you have done so, you can just delete your old new migration and make a new one, so that all changes are in one migration. Just make sure you have a fresh DB, before making the new migrations.